### DISCUSSION TITLE
Issue with the latest update release of Elementary 8

### DISCUSSION THREAD

**Discussion by inajmi** [elementary/triage#664] [Q&A] [answered]
after installing the latest updates available for elementary 8.0 the networking stops and it is not able to get an IP from the DHCP server nor will it work after defining a static IP address even. 

Please do let me know what logs do you require.

**Comment by ryonakano** [comment, marked as answer]
Could you try if https://github.com/elementary/switchboard-plug-network/issues/423#issuecomment-2554927909 work for you?

**Reply by inajmi** [reply]
this worked for me.

**Reply by ryonakano** [reply]
Leaving a note that this issue is fixed in https://github.com/elementary/os/pull/745. Starting from the next daily and stable ISOs this fix takes effect.
